sample-apps/communication/main.js

```diff
@@ -45,3 +50,51 @@ function onNavigateUsingMessageClick () {
   }
   veraApi.sendArmeMessage({ packageName: navigationPackageName, data })
 }
+
+async function onNearestPoiClick() {
+  const MAX_DISTANCE_V = 2
+  const MAX_DISTANCE_H = 3
+  var cameraPose = veraApi.getCameraPose()
+  const filter = {
+    bool: {
+      filter: [{
+        range: {
+          "ar:geometry.center_y": {
+            gte: cameraPose.translation[1] - MAX_DISTANCE_V,
+            lte: cameraPose.translation[1] + MAX_DISTANCE_V
+          }
+        }
+      }]
+    }
+  }
+  // Geo-filter array: camX, camZ, max 2D horizontal distance (m)
+  const geofilter = [cameraPose.translation[0], cameraPose.translation[2], MAX_DISTANCE_H]
+  const allNearbyPois = await veraApi.querySemanticObjects({confKey: appConfig._id, filter, geofilter})
```

Comment:
TODO: make this API clear

Reply:
Added TODO, will improve in the next PR

Reply:
In the way it is implemented now, it is necessarily ready when this is called.
In any case, I moved the initialization of appConfig to the beginning of the init() func, I think it's enough?